CommunityToolkit.Tooling.SampleGen/Metadata/ToolkitSampleButtonCommand.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Windows.Input;
+
+namespace CommunityToolkit.Tooling.SampleGen.Metadata;
+
+/// <summary>
+/// A command that invokes the provided <see cref="Action"/> when executed.
+/// </summary>
+public class ToolkitSampleButtonCommand : ICommand
```

Comment:
This left over from other PR? As this isn't working yet right? (Also causing warnings with unused events)